mopidy_musicbox_webclient/static/mb.manifest

```diff
@@ -1,6 +1,6 @@
 CACHE MANIFEST
 
-# 2016-01-31:v3
+# 2016-02-01:v1
 
 NETWORK:
 *
```

Comment:
I had a brief look around but could not find anything on an *easy* way to generate the manifest file automatically (without resorting to installing weird and wonderful add-ons and plugins).

Unless anybody has any ideas, I suggest we just keep this a manual step for now...